tests/test_reflection_views.py

```diff
@@ -30,13 +34,17 @@ def test_late_binding_view_reflection(redshift_engine):
     view_ddl = ("CREATE VIEW my_late_view AS "
                 "%s WITH NO SCHEMA BINDING" % view_query)
     conn = redshift_engine.connect()
-    conn.execute(table_ddl)
-    conn.execute(view_ddl)
-    insp = inspect(redshift_engine)
-    view_definition = insp.get_view_definition('my_late_view')
-
-    # For some reason, Redshift returns the entire DDL for late binding views.
-    assert(clean(compile_query(view_definition)) == clean(view_ddl))
-    view = Table('my_late_view', MetaData(),
-                 autoload=True, autoload_with=redshift_engine)
-    assert(len(view.columns) == 2)
+    try:
+        conn.execute(table_ddl)
+        conn.execute(view_ddl)
+        insp = inspect(redshift_engine)
+        view_definition = insp.get_view_definition('my_late_view')
+
+        # Redshift returns the entire DDL for late binding views.
+        assert(clean(compile_query(view_definition)) == clean(view_ddl))
+        view = Table('my_late_view', MetaData(),
+                     autoload=True, autoload_with=redshift_engine)
+        assert(len(view.columns) == 2)
+    finally:
+        conn.execute('DROP TABLE IF EXISTS my_table CASCADE')
+        conn.execute('DROP VIEW IF EXISTS my_late_view CASCADE')
```

Comment:
These try/finally blocks should no longer be necessary if we're correctly running tests in the build-specific database.

Reply:
I read over the PR again, and reverting this change in the tests is my last bit of feedback. We should be good to merge and make a release once this is reverted and we ensure all tests are passing.

Thanks so much for your efforts here. There will be many users glad to be able to use SQLA 1.4.

Reply:
Is there any reason that cleaning up is not wanted? It doesn't seem like this would do anything but protect against future tests having issues.

Reply:
Fair enough. I'll kick off final integration tests here.